x-pack/test/functional/apps/ml/settings/filter_list_edit.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+
+  const filterId = 'test_filter_list_edit';
+  const keywordToDelete = 'keyword_to_delete';
+  const oldDescription = 'Old filter list description';
+
+  const newKeywords = ['new_keyword1', 'new_keyword2'];
+  const newDescription = 'New filter list description';
+
+  describe('filter list edit', function () {
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await ml.api.createFilter(filterId, {
+        description: 'Old filter list description',
+        items: [keywordToDelete, 'old_keyword'],
```

Comment:
We should probably have a variable for `old_keyword` that can be used in a later validation, see below.

Reply:
Updated here 20c6b2d